### ISSUE TITLE
AppMain linker build errors

### ISSUE THREAD

**Issue by bobpbbd104** [microsoft/HoloLensForCV#102]
Running VS 2017 15.9 and get 2 errors:
18>AppMain.obj : error LNK2019: unresolved external symbol "class cv::debug_build_guard::_InputOutputArray const & __cdecl cv::noArray(void)" (?noArray@cv@@YAABV_InputOutputArray@debug_build_guard@1@XZ) referenced in function "class std::map<int,struct ArUcoMarkerTracker::DetectedMarker,struct std::less<int>,class std::allocator<struct std::pair<int const ,struct ArUcoMarkerTracker::DetectedMarker> > > __cdecl ArUcoMarkerTracker::DetectArUcoMarkers(class HoloLensForCV::SensorFrame ^)" (?DetectArUcoMarkers@ArUcoMarkerTracker@@YA?AV?$map@HUDetectedMarker@ArUcoMarkerTracker@@U?$less@H@std@@V?$allocator@U?$pair@$$CBHUDetectedMarker@ArUcoMarkerTracker@@@std@@@4@@std@@P$AAVSensorFrame@HoloLensForCV@@@Z)
18>AppMain.obj : error LNK2019: unresolved external symbol "void __cdecl cv::aruco::detectMarkers(class cv::debug_build_guard::_InputArray const &,struct cv::Ptr<class cv::aruco::Dictionary> const &,class cv::debug_build_guard::_OutputArray

**Comment by cde-git** [comment]
@bobpbbd104 : Did you ever resolve this issue ?

**Comment by bobpbbd104** [comment]
No, guess it’s an older Microsoft sample with little interest now, project not maintained. I made some HoloLens progress without this particular example.
